.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "SpongeAPI"]
 	path = SpongeAPI
-	url = https://github.com/SpongePowered/SpongeAPI.git
+	url = https://github.com/JBYoshi/SpongeAPI.git
```

Comment:
So I don't change those when I'm doing a pull request? Sorry. This is my first pull request with submodules, so I wasn't sure how to do that.